### PR TITLE
Se feedback 7

### PR DESCRIPTION
**Pull request by breijm**
Questions, Comments, Feedback - all are welcome
Last chance to make changes before the deadline Friday

### CODE REVIEW

File: se_feedback/sprint_7/feedback_sprint_7_all.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+|---------|-------|
+| Tooling |  10   |
+
+| Pull-based Development | Score |
+|------------------------|-------|
+| Branching              |  10   |
+| Code Review            |  10   |
+
+##Notes
+* Parsers have high cyclomatic complexity (ex. PMDXMLParser)
```
Comment (SunweiWang) [inline se_feedback/sprint_7/feedback_sprint_7_all.md]: Hi, Bastiaan, I have already split the method in the parsers, and by cyclomatic complexity, do you mean they are too many independent paths? I have done following fix: #95 , did I misunderstand what you mean or is it also a fix as well?

Comment (breijm) [inline se_feedback/sprint_7/feedback_sprint_7_all.md]: Hey @SunweiWang, #95 is a good fix; removing the if statements certainly helped.

**Comment by boninggong** [comment]
TA reviewed and approved, I think this is mergable now

File: se_feedback/sprint_7/feedback_sprint_7_all.md (hunk 1 above)
Comment (SunweiWang) [inline se_feedback/sprint_7/feedback_sprint_7_all.md]: Thanks, we can merge my PR then xD

**Pull request merged**